english/33hours/wakingup.md

```diff
@@ -5,6 +5,8 @@ cheap motel room. You have no idea where you are or how you got here.
 You sit up and immediately feel pain on the back of your head. Some dried blood cakes the pillow you were lying on. Your
 wallet and mobile phone are gone. In fact, your pockets are completely empty.
 
+[Read the news in Burmese.](burmese/burmese.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.